domain-models-runtime/src/main/java/org/folio/rest/tools/utils/JwtUtils.java

```diff
@@ -1,17 +1,48 @@
 package org.folio.rest.tools.utils;
 
-import java.io.UnsupportedEncodingException;
+import io.vertx.core.json.JsonObject;
+import java.nio.charset.StandardCharsets;
 import java.util.Base64;
 
 /**
  * @author shale
  *
  */
-public class JwtUtils {
+public final class JwtUtils {
+  private JwtUtils() {
+    throw new UnsupportedOperationException("Cannot instantiate utility class");
+  }
+
+  /**
+   * @return strEncoded decoded using Base64 and UTF-8.
+   */
+  public static String getJson(String strEncoded) {
+    return new String(Base64.getDecoder().decode(strEncoded), StandardCharsets.UTF_8);
+  }
 
-  public static String getJson(String strEncoded) throws UnsupportedEncodingException{
-    byte[] decodedBytes = Base64.getDecoder().decode(strEncoded);
-    return new String(decodedBytes, "UTF-8");
+  /**
+   * Get the value for key from the payload. It neither validates the
+   * token header nor the token signature.
+   * @param token  a JWT token
+   * @return the value for key as String, or null if the key does not exist or token cannot be decoded
+   */
+  public static String get(String key, String token) {
+    try {
+      if (token == null) {
+        return null;
+      }
```

Comment:
null check does not need to be in the try block

Reply:
JwtUtils.java has been deleted.